BUILD.md

```diff
@@ -69,7 +69,12 @@ If you get the error `libtool: unrecognized option '-static'`, follow the instru
 
 ## On Windows
 
-If node-gyp does not work (MSBUILD: error MSB3428: Could not load the Visual C++ component "VCBuild.exe"), you might need to install `windows-build-tools` using `npm install --global windows-build-tools`.
+There are various errors that can occur from an improper build environment (such as MSBUILD: error MSB3428). It is recommended to install `windows-build-tools` with the command `npm install --global windows-build-tools` (elevation required) and then using these two commands to set the environmental variables to the proper values:
+
+```batch
+call "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build\vcvarsamd64_x86.bat" 
+set "PATH=C:\Program Files\nodejs;%PATH%"
```

Comment:
I think that's a bit too specific for your particular environment. Visual C++ support should not be done by us.

Reply:
This batch file comes default with the `npm install --global windows-build-tools` command, I did not have it before I ran this command. This establishes many variables that the build process references for things like the proper toolset version to use for compiling sqlite3 and such.

Reply:
@laurent22 Here's the relevant line of code in windows-build-tools: https://github.com/felixrieseberg/windows-build-tools/blob/master/ps1/launch-installer.ps1#L37

And the included packages in the `Microsoft.VisualStudio.Workload.VCTools` workload can be seen here: https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2017#visual-c-build-tools